### ISSUE TITLE
Multiple inheritance

### ISSUE THREAD

**Issue by abeelen** [rsokl/custom_inherit#38]
When having multiple inheritance (without properly using [ABC](https://github.com/rsokl/custom_inherit#advanced-usage) or mixins #20 I found a strange behavior : 

```python
from custom_inherit import DocInheritMeta


class Parent(metaclass=DocInheritMeta(style="numpy_with_merge")):
    """Parent.

    Notes
    -----
    Blah
    """

    def meth(self, x, y=None):
        """
        Raises
        ------
        NotImplementedError"""
        raise NotImplementedError


class Child1(Parent):
    """Child1. 

    Methods
    -------
    meth: does something
    """

    def meth(self, x, y=None):
        """Method description. """
        return 0


class Child2(Parent):
    """Child 2.

    Methods
    -------
    blah : does not much
    """
    def blah(self):
        """Method blah."""
        pass

class GrandChild(Child1, Child2):
    """Grand Child."""
    pass
```

```python
>>> GrandChild.__doc__
'Grand Child.\n\nMethods\n-------\nblah : does not much\nmeth: does something\n\nNotes\n-----\nBlah\nBlah\nBlah\nBlah'
```

The doc of the grand parent class gets repeated. Is that the correct behavior ?

**Comment by rsokl** [comment]
Sorry about the delay on this. I believe the `Notes` section needs to be removed from the multi-merge-able sections introduced in #31. Furthermore, we need to add testing / documentation for the expected behavior with multiple inheritance.

**Comment by ickc** [comment]
I think I'm seeing the same bug. My example is a little different using v2.3.2:

```py

from dataclasses import dataclass

from custom_inherit import DocInheritMeta


@dataclass
class A(metaclass=DocInheritMeta(style='numpy_napoleon_with_merge')):
    """Testing A.

    Parameters
    ----------
    a :
        parameter a.
    b :
        parameter b.
    c :
        parameter c.

    Returns
    -------

    """
    a: str = 'a'
    b: str = 'b'
    c: str = 'c'


@dataclass
class B(A):
    """Testing B.

    Parameters
    ----------
    d :
        parameter d.
    e :
        parameter e.
    f :
        parameter f.

    Returns
    -------

    """
    d: str = 'd'
    e: str = 'e'
    f: str = 'f'


@dataclass
class C(B):
    """Testing C.

    Parameters
    ----------
    g :
        parameter g.
    h :
        parameter h.
    i :
        parameter i.

    Returns
    -------

    """
    g: str = 'g'
    h: str = 'h'
    i: str = 'i'
```

Run this with `python -c 'from testing import C; print(C.__doc__)'` shows

```rst
Testing C.

Parameters
----------
a :
    parameter a.
b :
    parameter b.
c :
    parameter c.
a :
    parameter a.
b :
    parameter b.
c :
    parameter c.
d :
    parameter d.
e :
    parameter e.
f :
    parameter f.
g :
    parameter g.
h :
    parameter h.
i :
    parameter i.

Returns
-------
```

where the a, b, c from the parent repeated.

**Comment by rsokl** [comment]
@ickc @abeelen this has been fixed in `v2.4.0` which is now available on pypi (and will be available on conda-forge within the next couple of days).

Apologies for these issues and for the slow response.

**Comment by ickc** [comment]
No problem at all and thanks for the updates. I'll test it soon and if there's further problem I'll let you know.
